markdown/chapters/entombsday/playing-the-deck.md

```diff
@@ -0,0 +1,214 @@
+---
+authors: drynne
+order: 1
+title: Playing the Deck
+---
+
+## Preamble
+
+As a deck with two unique gameplans, Entombsday requires some flexibility from its pilot. Knowing the ins-and-outs of both Tin Fins and Doomsday will improve your performance with the deck, as will developing your own heuristics for when to uses which plan.
+
+Because this Wiki already contains excellent resources for how to play Turbo Doomsday, we will primarily focus on the Tin Fins plan and when to *transform* into Doomsday.
+
+## Playing Tin Fins
+
+Fundamentally, Tin Fins is an A + B reanimator combo deck.
+
+### Lion's Eye Diamond in Tin Fins
+
+:card[Lion's Eye Diamond] is a tricky card to use properly due to the rules weirdness surrounding it. It is primarily included in the deck for its use in Doosday piles, but there are a few important interactions to be aware of for the Tin Fins half of the deck.
+
+LED can be used to *bin* a Griselbrand in response to a :card[Shallow Grave]. This unfortunately does not work with Emrakul, as her shuffle trigger will be stacked on top of Shallow Grave and leave you with an empty graveyard when Shallow Grave resolves.
+
+Similar to its use in Storm decks and Doomsday piles, LED can be cracked in response to a Griselbrand activation to make the mana available for the cards you will be drawing. Just make sure you hold priority when activating Griselbrand by holding Control on MTGO or announcing it in paper.
+
+### Infinite Loops
+
+In the event that you can’t find a way to reanimate Emrakul pre-combat, you are able to *loop* your deck to either cast Emrakul or repeatedly cast :card[Collective Brutality] to drain your opponent out 2 life at a time.
+
+To *loop* your deck:
+
+1. Draw 14 to 21 cards with Griselbrand until you can put :card[Children of Korlis] into play (either by casting or reanimating it).
+2. Sacrifice Children of Korlis to regain all of the life you’ve lost so far this turn (this effect is **cumulative**, so you can gain functionally infinite life if you *loop* Children).
+3. Using the life gain from Children, draw your entire library and begin casting Dark Rituals to generate a large amount of {B} mana.
```

Comment:
Be careful not to draw yourself to death.....
Maybe make a note of what do if not find children with 6 cards left etc (use LP and Pn to find them).

Reply:
Added these under new "general tips" section